FF1Lib/Sanity/NoYouDonTException.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace FF1Lib.Sanity
+{
+	[Serializable]
+	internal class NoYouDonTException : Exception
```

Comment:
Don't with a capital `T` really?

Reply:
I added this one, when i first realized there were 3500 teleporters in coneria. I'm sure we find a solution for the T.